private/buf/bufprotopluginexec/version_test.go

```diff
@@ -55,6 +56,33 @@ func TestGetFeatureProto3OptionalSupported(t *testing.T) {
 	assert.True(t, getFeatureProto3OptionalSupported(newVersion(21, 0, 0, "")))
 }
 
+func TestGetFeatureEditionsSupported(t *testing.T) {
+	t.Parallel()
+	assertEditionsNotSupported(t, newVersion(2, 12, 4, ""))
```

Comment:
nit: Maybe overkill since we'd only really need to test < 27 and >= 27 to start. I imagine these will get more interesting over time as new editions are supported.

Reply:
I left a couple of extra tests, just to sanity check the output from older versions of protoc and buf. Hopefully it's not too bad overkill.